posthog/models/special_migration.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+# an enum, essentially
+class MigrationStatus:
+    NotStarted = 0
+    Running = 1
+    CompletedSuccessfully = 2
+    Errored = 3
+    RolledBack = 4
+
+
+class SpecialMigration(models.Model):
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["name"], name="unique name",)]
+
+    id: models.BigAutoField = models.BigAutoField(primary_key=True)
+    name: models.CharField = models.CharField(max_length=400, null=False, blank=False)
```

Comment:
Nit: I know we don't expect this table to be huge in length but 400 looks a bit a waste to me. We could add an additional column `description` if we want to add more information to it

Reply:
Description is a good idea indeed. Will be very useful in the UI.

Reply:
Added description, limited to 50 + a description with 400 max